website/static/css/pages/remove-contributor-page.css

```diff
@@ -0,0 +1,4 @@
+.col-md-4 {
+    padding-top: 20px;
+    padding-right: 15px;
+}
```

Comment:
Yeah, no preprocessor at the moment. Only bundling (i.e. `require('my-css-module.css')`) for non-critical CSS. Thanks for reticketing.